src/services/study/loadflow.ts

```diff
@@ -15,7 +25,7 @@ export function getDefaultLoadFlowProvider() {
     return backendFetchText(getDefaultLoadFlowProviderUrl);
 }
 
-export function setLoadFlowParameters(studyUuid, newParams) {
+export function setLoadFlowParameters(studyUuid: UUID, newParams: any) {
```

Comment:
any should be replaced by a type

Reply:
or else `unknown` if we really don't know what it is.

Reply:
Done.